plugins/processors/date/date.go

```diff
@@ -0,0 +1,69 @@
+package date
+
+import (
+	"github.com/influxdata/telegraf"
+	"github.com/influxdata/telegraf/plugins/processors"
+)
+
+const sampleConfig = `
+##Specify the date tags to add
+tagKey = "month"
+dateFormat = "%m"
```

Comment:
Make these snake_case, 2 space indent so it will look right with `telegraf config`.